pkg/api/types.go

```diff
@@ -584,10 +584,36 @@ const (
 	ClaimLost PersistentVolumeClaimPhase = "Lost"
 )
 
+type HostPathType string
+
+const (
+	// If nothing exists at the given path, an empty directory will be created there.
+	HostPathUnset HostPathType = "DirectoryOrCreate"
```

Comment:
If you're leaving this first because it is default, that's OK, but put Directory right after it.